doc/advanced-configuration/acl.md

```diff
@@ -94,16 +97,17 @@ As an example we can discuss 2 rules which are present in the default config fil
 * `{access, register, [{allow, all}]}`.<br/>
 This rule is used while registering a new user, the example above has no restrictions, but we might want to block certain JIDs, `admin` JID for instance.
```

Comment:
This rule is used while registering a new user. The above example has no restrictions, but...

Reply:
ok